rcsb/utils/targets/CARDTargetAnnotationProvider.py

```diff
@@ -0,0 +1,213 @@
+                descr = cardP.getModelValue(modelId, "descr")
+                cvTermId = cardP.getModelValue(modelId, "cvTermId")
+                annotationId = "ARO:" + cardP.getModelValue(modelId, "accession")
+                annotationLineage = cardP.getLineage(annotationId)
+                familyCvTermId = cardP.getModelValue(modelId, "familyCvTermId")
+                familyName = cardP.getModelValue(modelId, "familyName")
+                familyAnnotationId = "ARO:" + cardP.getModelValue(modelId, "familyAccession")
+                familyDescription = cardP.getModelValue(modelId, "familyDescription")
+                drugClasses = cardP.getModelValue(modelId, "drugClasses")
+                resistanceMechanism = cardP.getModelValue(modelId, "resistanceMechanism")
+                familyAnnotationLineage = cardP.getLineage(familyAnnotationId)
+                rD = {
+                    "entry_id": entryId,
+                    "entity_id": entityId,
+                    # "type": "CARD",  # Assigned in DictMethodEntityHelper
+                    # "provenance_source": "matching CARD Protein Homolog Models (PHM)",  # Assigned in DictMethodEntityHelper
+                    "name": nm,
+                    "annotation_id": annotationId,
+                    "card_aro_cvterm_id": cvTermId,
+                    "description": descr,
+                    "seq_id_pct": seqIdPct,
+                    "bit_score": bitScore,
+                    "reference_scheme": refScheme,
+                    "assignment_version": assignVersion,
+                    "annotation_lineage": annotationLineage,
+                    "query_tax_name": matchD["queryTaxName"] if "queryTaxName" in matchD else None,
+                    "target_tax_name": matchD["targetTaxName"] if "targetTaxName" in matchD else None,
+                    "match_status": matchD["taxonomyMatchStatus"] if "taxonomyMatchStatus" in matchD else None,
+                    "lca_tax_name": matchD["lcaTaxName"] if "lcaTaxName" in matchD else None,
+                    "family_annotation_id": familyAnnotationId,
+                    "family_card_aro_cvterm_id": familyCvTermId,
+                    "family_name": familyName,
+                    "family_description": familyDescription,
+                    "card_aro_drug_class": drugClasses,
+                    "card_aro_resistance_mechanism": resistanceMechanism,
+                    "family_annotation_lineage": familyAnnotationLineage,
+                }
+                rDL.append(rD)
+        #
+        qD = {}
+        dD = {}
+        for rD in rDL:
+            eId = rD["entry_id"] + "_" + rD["entity_id"]
+            fId = rD["annotation_id"]
+            if (eId, fId) in dD:
+                continue
+            dD[(eId, fId)] = True
+            qD.setdefault(eId, []).append(rD)
+        # --
+        # If useTaxonomy filter is True, resulting dictionary contains ~50 fewer perfect matches (as of March 2023)
```

Comment:
I don't follow the reason for this `useTaxonomy` option? Is it actually used?

Reply:
By default it was set to off, but in `ProteinTargetSequenceExecutionWorkflow.py` it was turning it on (https://github.com/rcsb/py-rcsb_workflow/blob/master/rcsb/workflow/targets/ProteinTargetSequenceExecutionWorkflow.py#L168). I'll need to talk with Yana about whether we want to use it or not, but since we get more "perfect" matches with it turned off I'm leaning toward that.

Reply:
Make sense, thanks.